src/riak_client.erl

```diff
@@ -878,7 +879,35 @@ ttaaefs_fullsync(WorkItem, SecsTimeout, Now) ->
     wait_for_reqid(ReqId, SecsTimeout * 1000).
 
 
-
+-spec remove_node_from_coverage() -> ok.
+remove_node_from_coverage() ->
```

Comment:
If you give an argument to this function ParticipateInCoverage, then the second one can call the first with the default value. Less code, more consistency